website/src/pages/Status/Status.jsx

```diff
@@ -15,12 +15,133 @@ limitations under the License.
 */
 
 import React from 'react';
+import { useState, useEffect } from 'react';
+
+import './status.css'
+
+import PreviousExe from '../../components/PreviousExecutions/PreviousExe';
+import ExeQueue from '../../components/ExecutionQueue/ExeQueue';
+import RingLoader from "react-spinners/RingLoader";
 
 const Status = () => {
+
+  const [isLoading, setIsLoading] = useState(true)
+  const [dataQueue, setDataQueue] = useState([]);
+  const [dataPreviousExe, setDataPreviousExe] = useState([]);
+  
+  
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const responseQueue = await fetch('http://localhost:9090/api/queue');
+        const responsePreviousExe = await fetch('http://localhost:9090/api/recent');
+  
+        const jsonDataQueue = await responseQueue.json();
+        const jsonDataPreviousExe = await responsePreviousExe.json();
+        
+        setDataQueue(jsonDataQueue);
+        setDataPreviousExe(jsonDataPreviousExe);
+        setIsLoading(false)
+      } catch (error) {
+        console.log('Erreur lors de la récupération des données de l\'API', error);
```

Comment:
Error should be in english. Moreover, we should ideally have some kind of fallback if the API fails. Instead of leaving the loader loading forever, we should show some text on the page informing the user that the load failed and inviting the user to try again.